src/protagonist/API.Tests/Integration/AssetTests.cs

```diff
@@ -683,4 +683,67 @@ public async Task Post_ImageBytes_Ingests_New_Image()
         asset.Origin.Should().Be("https://protagonist-test-origin.s3.eu-west-1.amazonaws.com/99/1/Post_ImageBytes_Ingests_New_Image");
 
     }
+
+    [Fact]
```

Comment:
Is this better as?

```cs
[Theory]
[InlineData("/customers/99/spaces/377/images?pageSize=50&q={\"string3\": \"16-20\"}", 5)]
[InlineData("/customers/99/spaces/377/images?pageSize=50&q={\"string2\": \"1-10\"}", 10)]
...
```

Having them as separate cases would make it easier to see at a glance if, e.g. 2 of the 5 fail. As is the whole case will fail if any of them fail - so could be 1 or all of them.

Could add the images in ctor, or append something to the Id's to make them unique per inline data.